gradle/ge.gradle

```diff
@@ -27,6 +31,25 @@ gradleEnterprise {
         obfuscation {
             ipAddresses { addresses -> addresses.collect { address -> "0.0.0.0"} }
         }
+
+        tag("JDK" + System.getProperty("java.version").find(/\d+/)) // major version
+        value("Java Vendor", System.getProperty("java.vendor"))
+        value("Java Version", System.getProperty("java.version"))
+
+        // Jenkins job name, less redundant "Solr" parts
+        String jenkinsJobName = System.getenv("JOB_NAME")
+        if (jenkinsJobName) { // looks like "Solr/Solr-Smoketest-9.5"
+            tag(jenkinsJobName.replaceAll(/Solr\W/,"")) // like "Smoketest-9.5"
```

Comment:
The Common Custom User Data Gradle Plugin already adds this as a custom value called `CI job` (but without the filtering you've added).

Reply:
Wanted to keep tags short